frontend/android/app/build.gradle

```diff
@@ -38,7 +38,7 @@ idea {
 ext.setupApplication = { ApplicationProductFlavor flavor, buildConfigName ->
     var buildConfig = createBuildConfig(buildConfigName)
     flavor.applicationId buildConfig.applicationId
-    flavor.manifestPlaceholders = [*:determineIcon(buildConfig)]
+    flavor.manifestPlaceholders = [*:determineAppName(buildConfig), *:determineIcon(buildConfig)]
```

Comment:
Could we just declare the manifest placeholders here explicitly, instead of calling several functions? We do not use those anywhere else.